tests/unit/src/util/environment.ts

```diff
@@ -0,0 +1,191 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import * as environment from '../../../../src/util/environment';
+import loadModule, { cleanupModuleMocks } from '../../../_support/loadModule';
+import { stub, SinonStub } from 'sinon';
+
+const relevantEnv = [
+	'TRAVIS_COMMIT_MESSAGE',
+	'TRAVIS_BRANCH',
+	'ENCRYPTED_KEY_FILE',
+	'TRAVIS_COMMIT',
+	'HAS_GIT_CREDENTIALS',
+	'HEXO_ROOT',
+	'TRAVIS_EVENT_TYPE',
+	'KEY_FILE',
+	'DEPLOY_DOCS',
+	'PUBLISH_TARGET_REPO',
+	'TRAVIS_REPO_SLUG'
+];
+const file = 'test.file';
+
+let mappedEnvs: { name: string; value: string; }[];
+
+let module: any;
+let existsSyncStub: SinonStub;
+
+registerSuite({
+	name: 'util/environment',
+
+	before() {
+		mappedEnvs = relevantEnv.map((name) => ({ name, value: process.env[name] }));
+
+		existsSyncStub = stub();
+	},
+
+	after() {
+		cleanupModuleMocks();
+	},
+
+	beforeEach() {
+		module = loadModule('src/util/environment', {
+			fs: {
+				existsSync: existsSyncStub
+			}
+		}, false);
+	},
+
+	afterEach() {
+		mappedEnvs.forEach((val) => {
+			if (val.value) {
+				process.env[val.name] = val.value;
+			} else {
+				delete process.env[val.name];
+			}
+		});
+
+		existsSyncStub.reset();
+	},
+
+	commitMessage() {
+		let commit = 'update test coverage for `util/environment`';
+		process.env.TRAVIS_COMMIT_MESSAGE = commit;
+		assert.equal(environment.commitMessage(), commit);
+	},
+
+	currentBranch() {
+		let branch = 'master';
+		process.env.TRAVIS_BRANCH = branch;
+		assert.equal(environment.currentBranch(), branch);
+	},
+
+	decryptIvName() {
+		assert.equal(environment.decryptIvName(), 'publish_deploy_iv');
+	},
+
+	decryptKeyName() {
+		assert.equal(environment.decryptKeyName(), 'publish_deploy_key');
+	},
+
+	encryptedKeyFile() {
+		let filename = 'keyfile';
+		let fileWithExt = filename + '.enc';
+		let keyFileDefault = 'deploy_key.enc';
+
+		delete process.env.ENCRYPTED_KEY_FILE;
+		delete process.env.KEY_FILE;
+		assert.equal(environment.encryptedKeyFile(), keyFileDefault);
+
+		process.env.KEY_FILE = filename;
+		assert.equal(environment.encryptedKeyFile(), fileWithExt);
```

Comment:
we don't need this test. The same branch logic for `encryptedKeyFile()` is already tested in the above test.